contracts/modules/Clusters.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
```

Comment:
not related to this file but, can you move the test contracts to another folder? so we can skip them from generating the npm package, etc?
Also, prefix/suffix them to quickly see they are test files.